### DISCUSSION TITLE
Lava Installation failed

### DISCUSSION THREAD

**Discussion by Rashedape** [lava-nc/lava#250] [Q&A]
Hello,
I am trying to install Lava on my INRC cloud account and I am following the Lava (https://lava-nc.org/) suggested poerty-based approach. Unfortunately, I am not able to complete the installation process. After the poetry installation command, I am getting the following error:
 • Updating scipy (1.7.3 -> 1.8.0): Failed
  EnvCommandError
..........(some warning message)
        Successfully uninstalled scipy-1.7.3
  ERROR: Could not install packages due to an OSError: [Errno 16] Device or resource busy: '.nfs0000000007045c6f0000016b'

**Comment by awintel** [comment]
Thanks for your feedback. Best to file an issue in case of such problems.
